contracts/OXO.sol

```diff
@@ -0,0 +1,90 @@
+pragma solidity 0.4.15;
+
+import "./OxoFactory.sol";
+
+contract OXO is OxoFactory{
+
+ 	event StepEvent(address player, uint fileld, State state);
+ 	event GameOverEvent(State state);
+	event TransferEvent(string _text, address _to, uint _value);
+
+	 function doStep(uint _field) stillCanPlay() onlyWaitThisPlayer() onlyEmptyCell(_field) public{
+	   Game memory game = gameById[getGameId()];
+		game.board[_field] = (msg.sender == game.playerOne)? uint(keccak256("x")) : uint(keccak256("o"));
+		State _state = game.state;
+		game.countOfMove ++ ;
+		if(game.countOfMove >= 5){
+		    _state = checkWinner(game, _field);
+			if(_state != State.WinnerPlayerOne && _state != State.WinnerPlayerTwo ){
+		    	_state = checkDraw(game);
```

Comment:
Does it make sense to check for a draw when countOfMove < 9?

Reply:
inside 'checkDraw(game)' was the condition that checking execute when the count of steps was equal 9.
Nonetheless, you are right, as result, I have changed logic checking 'Draw', and add the condition that checks 'Draw' if a count of move more or equal 8

Reply:
Please  , Look again